cmd/mothership/mothership/start/http.go

```diff
@@ -127,11 +127,6 @@ func startWebserver(ctx context.Context, o *Options) error {
 		callHandler(o, createOrUpdateCluster)).
 		Methods(http.MethodPost, http.MethodPut)
 
-	apiRouter.HandleFunc(
```

Comment:
This for sure is not intended right?

Reply:
actually intended, if you check the block above, it's duplicated. I'm really questioning the review process of other team...